src/components/ConditionalFilter/UI/constrains.ts

```diff
@@ -1,7 +1,8 @@
 import { Row } from "./types";
 
 export const getItemConstraint = (constraint: Row["constraint"]) => ({
-  disableRemoveButton: !constraint?.removable,
+  // eslint-disable-next-line @typescript-eslint/key-spacing, @typescript-eslint/no-unnecessary-boolean-literal-compare
```

Comment:
why don't fix typings? if here lads something different than boolean, then cast in in upper component